rxbroadcast/src/main/java/com/cantrowitz/rxbroadcast/BroadcastRegistrar.java

```diff
@@ -7,22 +7,23 @@
 /**
  * Created by adamcantrowitz on 9/1/15.
  */
-class GlobalBroadcastProvider extends BroadcastProviderStrategy {
+class BroadcastRegistrar implements BroadcastRegistrarStrategy {
 
     protected final Context context;
+    private final IntentFilter intentFilter;
 
-    GlobalBroadcastProvider(IntentFilter intentFilter, Context context) {
-        super(intentFilter);
+    BroadcastRegistrar(IntentFilter intentFilter, Context context) {
```

Comment:
Oh, this is just super nit-picky, but the params above are in the order Context, IntentFilter but the constructor goes in the opposite order.

This is some serious bike shedding so feel free to ignore.